src/common/prisma.ts

```diff
@@ -108,9 +109,15 @@ export function isKeyValue(value: Value | KeyValue): value is KeyValue {
   return false;
 }
 
-export function assertValueIsStringArray(value: Value): Array<string> {
```

Comment:
I renamed this function to `assertValueIsArray` as technically speaking this function does not check whether the value is an array of strings, it just checks whether this is an array or not.

In `src/rules/require-field-index.ts` this correctly returns the array when the index is defined as `@@index([stringValue, functionValue(sort: Desc)]` then a type check is done on the return value